chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -401,7 +383,7 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
         // Ghost chat scenarios
         /* COMMENTING THIS CODE FOR PARITY WITH OLD LCW
         if (state.appStates.conversationState === ConversationState.Active &&
-            props.controlProps?.skipChatButtonRendering === true) {
+            props.controlProps?.hideStartChatButton === true) {
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             
             window.onbeforeunload = function () {
```

Comment:
what is the purpose of this?

Reply:
First `skipChatButtonRendering` is renamed for simplicity. Second, all other props in the interface starts with word `hide`, this one was different.